docs/topics/media-pipeline.rst

```diff
@@ -442,7 +442,7 @@ See here the methods that you can override in your custom Files Pipeline:
         successfully or ``False`` if it failed for some reason
 
       * ``file_info_or_error`` is a dict containing the following keys (if success
-        is ``True``) or a `Twisted Failure`_ if there was a problem.
+        is ``True``) or a :class:`twisted.python.failure.Failure` object if there was a problem.
```

Comment:
:lipstick: 
```suggestion
        is ``True``) or a :class:`~twisted.python.failure.Failure` object if there was a problem.
```

(_if_ you want it to show up as just `Failure`)

Reply:
If you don't mind, I'd prefer to keep the full class path. It's good to know that syntax trick tough :wink: